src/clr-angular/layout/_login.clarity.scss

```diff
@@ -93,6 +93,7 @@
             & > .clr-input-group {
               max-width: 100%;
               width: 100%;
+              padding-right: 0.4rem;
```

Comment:
Does 0.4rem align with our baseline? As it seems to be very specific, do we need to assign this to a variable?